arelle/CntlrCmdLine.py

```diff
@@ -289,6 +289,9 @@ def parseArgs(args):
     parser.add_option("--formulaVarExpressionResult", "--formulavarexpressionresult", action="store_true", dest="formulaVarExpressionResult", help=_("Specify formula tracing."))
     parser.add_option("--formulaVarFilterWinnowing", "--formulavarfilterwinnowing", action="store_true", dest="formulaVarFilterWinnowing", help=_("Specify formula tracing."))
     parser.add_option("--formulaVarFiltersResult", "--formulavarfiltersresult", action="store_true", dest="formulaVarFiltersResult", help=_("Specify formula tracing."))
+    parser.add_option("--testcaseFilter", "--testcasefilter", action="append", dest="testcaseFilters",
```

Comment:
arbitrary line:

Nice work! Given that we have access to more nodes now, how do you feel about removing the double approval requirement for conformance suite runs? I know I'm tired of waiting for find tests to complete just to approve a workflow a second time 😅